module.go

```diff
@@ -51,9 +51,10 @@ func init() {
 func (config *DistanceDetectorConfig) Validate(path string) ([]string, []string, error) {
 	deps := []string{}
 	var optionalDeps []string
-	if config.DefaultCamera != "" {
-		deps = append(deps, config.DefaultCamera)
+	if config.DefaultCamera == "" {
+		return nil, optionalDeps, errors.Errorf(`expected "camera_name" attribute (DefaultCamera) for obstacles pointcloud at %q`, path)
```

Comment:
I'd remove the part about DefaultCamera: only the "camera_name" is visible to the user. I'm also unsure about mentioning the path at the end: I suspect that's not typical of error messages from validation.